docs/api/wanchain.md

```diff
@@ -1,5 +1,255 @@
 # API / Wanchain
 
+## Deploy gateway
+
+Deploy gateway allows for delegating deployment of an `AssetLedger` smart contract to a third party and transfering value in the same atomic transaction.
+
+To perform a delegate deploy through `DeployGateway`, you need to follow its flow:
+
+1. Maker (address creating the delegate deploy) defines the deploy (defines the data for the `AssetLedger` that will be created, who will be the owner of that `AssetLedger` and what amount of tokens will be sent to what address in exchange).
```

Comment:
1. Maker (address creating the delegate deploy) defines the deploy by defining the data for the `AssetLedger` that will be created, the owner of that `AssetLedger`, the number of tokens sent in exchange, and the recipient address.